src/zenml/integrations/skypilot/flavors/skypilot_orchestrator_base_vm_config.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Skypilot orchestrator base config and settings."""
+
+from typing import Dict, Literal, Optional, Union
+
+from zenml.config.base_settings import BaseSettings
+from zenml.logger import get_logger
+from zenml.orchestrators import BaseOrchestratorConfig
+
+logger = get_logger(__name__)
+
+
+class SkypilotBaseOrchestratorSettings(BaseSettings):
+    """Skypilot orchestrator settings.
+
+    Attributes:
+        instance_type: the instance type to use.
+        cpus: the number of CPUs required for the task.
+            If a str, must be a string of the form ``'2'`` or ``'2+'``, where
```

Comment:
Just out of curiosity, why are double back ticks used here? What does that do actually?

Reply:
Good question, am not sure either. I think @htahir1 may have an answer for this

Reply:
Im also not sure i think i copied it from the source docs!